modules/dlscript/godot.cpp

```diff
@@ -180,6 +181,16 @@ void GDAPI *godot_dlinstance_get_userdata(godot_object *p_instance) {
 	return NULL;
 }
 
+void GDAPI godot_get_global_constants(godot_dictionary *p_constants) {
```

Comment:
Why not just return a dictionary? You can use `godot_dictionary_new()` and then return that.